flask_graphql/graphqlview.py

```diff
@@ -142,10 +143,10 @@ def get_response(self, request, data, show_graphiql=False):
     def json_encode(self, request, d, show_graphiql=False):
         pretty = self.pretty or show_graphiql or request.args.get('pretty')
         if not pretty:
-            return json.dumps(d, separators=(',', ':'))
+            return json.dumps(d, separators=(',', ':'), cls=self.json_encoder)
```

Comment:
I'm not an expert on json encoding/decoding in Python. What you've said sounds great but I'd rather leave this to someone in a better position to comment.